radio/src/targets/taranis/hal.h

```diff
@@ -1500,13 +1500,14 @@
 #elif defined(RADIO_X7ACCESS)
   #define INTMODULE_HEARTBEAT
   #define INTMODULE_HEARTBEAT_REUSE_INTERRUPT_ROTARY_ENCODER
-  #define INTMODULE_HEARTBEAT_RCC_AHB1Periph      RCC_AHB1Periph_GPIOA
-  #define INTMODULE_HEARTBEAT_GPIO                GPIOA
-  #define INTMODULE_HEARTBEAT_GPIO_PIN            GPIO_Pin_7  // PA.07
-  #define INTMODULE_HEARTBEAT_EXTI_PortSource     EXTI_PortSourceGPIOA
+  #define INTMODULE_HEARTBEAT_RCC_AHB1Periph      RCC_AHB1Periph_GPIOC
```

Comment:
I think you will fix the Trainer input on external module, but in the same time you break the internal module heartbeat which is used to improve the latency.

I need the schematics to fix it properly, will get back to this soon!

Reply:
@bsongis did you check whether or not on that radio the heartbeat input is share with the rotary encoder? If this is the case, aren't the defs here bogus anyway?

Reply:
Is this going to be fixed?
The problem is that I want to connect a ESP32 to the input, and I can find many libraries to generate a SBUS signal, but no for CPPM. (Does this bug also affect the input of CPPM-signals?)